weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/heuristics.ts

```diff
@@ -4,6 +4,10 @@ export const PREDICT_AND_SCORE_OP_NAME_POST_PYDANTIC =
 const EVALUATE_OP_NAME_PRE_PYDANTIC = 'Evaluation-evaluate';
 export const EVALUATE_OP_NAME_POST_PYDANTIC = 'Evaluation.evaluate';
 
+export const SUMMARIZE_OP_NAME = 'Evaluation.summarize';
+export const PREDICT_OP_NAME = 'Model.predict';
```

Comment:
why does predict have `Model.` prefex, but score does not have `Scorer.` ... this smells like a bug